tests/targets_tests.py

```diff
@@ -164,3 +165,33 @@ def test_s3_typed_uncompressed_csv_target(self):
         s.write_csv(df, index=False)
         read_result = s.read_csv()
         self.assertDictEqual(df.to_dict(), read_result.to_dict())
+
+    @mock_s3
+    def test_local_exists(self):
+        with TempDirectory() as d:
```

Comment:
Can you add a comment here explaining that "local_exists returns True even if target exists in S3"?